Tasks/Shchura/shchura_hometask1.py

```diff
@@ -0,0 +1,10 @@
+from decimal import Decimal
+deposit = Decimal(20000)
+months = 60
+interest_rate = Decimal(0.15)
+
+while months >  0:
+    deposit = deposit + ((deposit * interest_rate)/12)
+    months -= 1
```

Comment:
an overkill for this type of a task, there is a specific formula to calculate such percentage